modules/components/org.wso2.carbon.appfactory.provisioning.runtime/src/main/java/org/wso2/carbon/appfactory/provisioning/runtime/KubernetesPovisioningConstants.java

```diff
@@ -35,9 +35,13 @@
     public static final String KIND_DEPLOYMENT = "Deployment";
     public static final String KIND_INGRESS = "Ingress";
     public static final String KIND_SERVICE = "Service";
+    public static final String KIND_SECRETS = "Secret";
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/commits/7d70ba5f5bbacfe7789db67072db49c2ff893b78